storage/storage_local.go

```diff
@@ -0,0 +1,49 @@
+package storage
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/cavaliercoder/grab"
+	"github.com/specialedge/hangar-api/events"
+)
+
+type storageLocal struct {
+	Path string
+}
+
+func NewStorageLocal() Storage {
+	return &storageLocal{
+		Path: `F:\Code\specialedge\storage\java\`,
```

Comment:
This seems a bit hardcoded/specific for what is a more generically named package. Strip off the `java` and let each api/package type add their own subdirectory to this path? Or maybe even make this a configurable option, and still let each package type set their own subdirectory of this.

Reply:
Agreed, we want to make this configurable. Raised #2 for this.